tests/dualtor/test_normal_op.py

```diff
@@ -12,11 +12,6 @@
 ]
 
 
-@pytest.fixture(scope="session", autouse=True)
-def common_setup_teardown(run_arp_service):
```

Comment:
I think we never had `run_arp_service` fixture. Is that why this is now being removed?

Reply:
Yes, I believe it was a typo introduced in a previous PR